contentgrid-appserver-actuators/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java-library'
+    id 'io.freefair.lombok'
```

Comment:
Needs maven-publish plugin (and a description), because it needs to be published if we want to depend on it